src/html-conventions-lib/general-connectors-html-convention.xsl

```diff
@@ -541,17 +541,17 @@
         <xsl:sequence
             select="
                 if ($numberOfTags > 0) then
-                    f:generateHtmlWarning(fn:concat('The connector ', f:getConnectorName($connector), ' has tag annotations.', 
-            'The connector is not transformed into a property and therefore any tag will be ignored.'))
+                    f:generateHtmlWarning(fn:concat('The connector ', f:getConnectorName($connector), ' has tag annotations.',
```

Comment:
Why is this here? What does it mean? Not sure if I understand the logics.

Reply:
This is checking if a connector has tags. this is connector-tag-18 checker as per documentation.

Requirement of the checker: Check if connector has tags, raise a warning

Reply:
This is checking if a connector has tags. this is connector-tag-18 checker as per documentation.

Requirement of the checker: Check if connector has tags, raise a warning